src/inmanta/protocol/methods_v2.py

```diff
@@ -1775,3 +1775,14 @@ def graphql_schema() -> dict[str, Any]:
     To actually execute a query, use 'POST /api/v2/graphql'
     """
     pass
+
+
+@typedmethod(path="/health", operation="GET", client_types=[ClientType.api], enforce_auth=False, api_version=2)
+def health() -> ReturnValue[None]:
+    """
+    Returns a 200 response code if the server is healthy
+    or a 500 if the server is not healthy.
```

Comment:
500 Internal Server Error? That's unconventional, isn't it?

Reply:
on K8S livenes probes (which are the best reference I have for this), 

Any code greater than or equal to 200 and less than 400 indicates success. Any other code indicates failure.

the example there also uses 500 to indicate failure 

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
I didn't really find a clear answer on what is conventional. Some applications return 500 or 503 for not healthy. Others return a 200 and mention the status in the json body. What do you consider the conventional way?